packages/internal-scripts/src/release-image.ts

```diff
@@ -12,47 +12,90 @@ import {readFile} from 'fs/promises';
 import path from 'path';
 
 const optionDefinitions = [
-  {name: 'file', type: String, defaultOption: true},
-  {name: 'version', type: String},
+  {
+    name: 'files',
+    alias: 'f',
+    type: String,
+    defaultOption: true,
+    multiple: true,
+  },
+  {name: 'versions', alias: 'v', type: String, multiple: true},
 ];
 
+/**
+ * A cache of the parsed changelogs such that a package may be referenced
+ * multiple times to render multiple versions.
+ */
+const CHANGELOG_CACHE = new Map<string, Changelog>();
+
 export const run = async () => {
   const options = commandLineArgs(optionDefinitions);
 
-  if (!options.file) {
+  if (
+    !options.files ||
+    !Array.isArray(options.files) ||
+    (Array.isArray(options.versions) &&
+      options.versions.length !== options.files.length)
+  ) {
     console.error(
       `
 USAGE
-    release-image CHANGELOG_PATH [--version VERSION]
+    release-image CHANGELOG_PATH
+    release-image (-f CHANGELOG_PATH [-v VERSION])...
 
 EXAMPLES
     To generate the release image for the reactive-element package:
 
         release-image packages/reactive-element/CHANGELOG.md
+
+    For multiple packages in a single image:
+
+        release-image reactive-element/CHANGELOG.md lit-html/CHANGELOG.md
+
+    To generate an image composed of specific version numbers, including
+    multiple versions of the same package:
+
+        release-image -f reactive-element/CHANGELOG.md -v 3.2.0 \\
+                      -f lit-html/CHANGELOG.md -v 2.0.1 \\
+                      -f lit-html/CHANGELOG.md -v 2.0.0
+
           `.trim()
     );
     process.exit(1);
   }
-  const filename: string = options.file;
-  const version: string = options.version;
 
-  const packageJson = JSON.parse(
-    await readFile(path.join(path.dirname(filename), 'package.json'), {
-      encoding: 'utf-8',
-    })
-  );
-  const packageName = packageJson.name;
-
-  console.log(`Reading ${packageName} release ${version} from ${filename}`);
-  const changelog = (await parseChangelog({
-    filePath: filename,
-    removeMarkdown: false,
-  })) as Changelog;
-  const release = await getRelease(changelog, version);
-  if (release === undefined) {
-    throw new Error('no release found');
+  const releasesToRender: Release[] = [];
+  for (let i = 0; i < options.files.length; i++) {
+    const filename: string = options.files[i];
+    const version: string = (options.versions ?? [])[i];
+    let changelog = CHANGELOG_CACHE.get(filename);
+    if (!changelog) {
+      const packageJson = JSON.parse(
+        await readFile(path.join(path.dirname(filename), 'package.json'), {
+          encoding: 'utf-8',
+        })
+      );
+      const packageName = packageJson.name as string;
+
+      console.log(`Reading ${packageName} release ${version} from ${filename}`);
```

Comment:
A minor, totally unnecessary, suggestion. I found it jarring to see "release undefined" get logged when this runs without a version.
```suggestion
      console.log(`Reading ${packageName} release ${version ?? 'latest'} from ${filename}`);
```